openmetadata-ui/src/main/resources/ui/src/pages/UserListPage/UserListPageV1.tsx

```diff
@@ -172,6 +174,10 @@ const UserListPageV1 = () => {
   const handleSearch = (value: string) => {
     setSearchValue(value);
     setCurrentPage(INITIAL_PAGING_VALUE);
+    history.replace({
+      pathname: location.pathname,
+      search: value,
+    });
```

Comment:
Why are we not using push?
for every search, we want to store the URL in the visited location array right?

Reply:
The function is getting called for onChange event, so pushing routes will unnecessarily push multiple routes in the history. That's why I used replace here. I will add the comment here